### PR TITLE
Make UOPZ compatible with XDebug again

### PR DESCRIPTION
**Pull request by SMillerDev**
As suggested by @cmb69 in https://github.com/krakjoe/uopz/issues/122#issuecomment-605967700

I have no idea about the C code so I might need someone to point me to problems here.

### CODE REVIEW

**Comment by remicollet** [comment]
Better to also add a constraint in the package.xml

```
diff --git a/package.xml b/package.xml
index 8dd9afc..43ac313 100644
--- a/package.xml
+++ b/package.xml
@@ -134,6 +134,11 @@ It supports the following activities:
    <pearinstaller>
     <min>1.10</min>
    </pearinstaller>
+   <extension>
+    <name>xdebug</name>
+    <max>2.9.3</max>
+    <conflicts />
+   </extension>
   </required>
  </dependencies>
  <providesextension>uopz</providesextension>

```

This will result in following error message
`pecl/uopz conflicts with PHP extension "xdebug" (version <= 2.9.3), installed version is 2.9.3`

**Comment by SMillerDev** [comment]
Done.

File: uopz.c [hunk 1]
```diff
@@ -24,7 +24,11 @@
 #include "php_ini.h"
 #include "ext/standard/info.h"
 
-#include "SAPI.h"
+#ifndef HAVE_SPL
+/* {{{ */
+zend_class_entry *spl_ce_RuntimeException;
+zend_class_entry *spl_ce_InvalidArgumentException; /* }}} */
+#endif
```
Comment (remicollet) [inline uopz.c:31]: Please don't readd this SPL thing, not needed

Comment (SMillerDev) [reply uopz.c]: Fixed (did I mention I had no idea about the C code)

**Comment by SMillerDev** [comment]
Anything I can still do here to help?

**Comment by cmb69** [comment]
Would be great if @krakjoe could review this PR.

**Comment by SMillerDev** [comment]
Can this be merged without that review?

**Comment by krakjoe** [comment]
@cmb69 merge it please
@remicollet I think probably a release is in order for this one, please

**Comment by remicollet** [comment]
Merged

**Comment by SMillerDev** [comment]
Thanks for all the work people

**Comment by remicollet** [comment]
>  @remicollet I think probably a release is in order for this one, please

@krakjoe 6.1.2 released

BTW, can you pleasde have a look at travis results, seems we have some failure when opcache is enabled (7.4 only)

**Comment by krakjoe** [comment]
Thanks && ack

**Pull request closed without merging**